pages/dkp/kommander/2.2/dkp-upgrade/index.md

```diff
@@ -1,10 +1,62 @@
 ---
 layout: layout.pug
-navigationTitle: DKP upgrade
-title: How to upgrade DKP
+navigationTitle: Upgrade DKP
+title: How to upgrade DKP via CLI
 menuWeight: 8
-excerpt: Upgrade DKP to the latest version and benefit from the full DKP experience.
+excerpt: Upgrade DKP to the latest version
 beta: false
 ---
 
-This section describes how to upgrade your environment to the latest version of DKP.
+The DKP upgrade represents an important step of your environment's lifecycle, as it ensures that you are up-to-date with the latest features and can benefit from the latest improvements, enhanced cluster management, and better performance. This section describes how to upgrade your networked, air-gapped or on-prem environment to the latest version of DKP.
+
+## Understand the upgrade process
+
+For this release, you perform the upgrade sequentially beginning with DKP Kommander and then moving to DKP Konvoy.
+
+When upgrading DKP, the process is different depending on if you run a stand-alone Management Cluster, or if you run a multi-cluster environment that includes a combination of a Management Cluster and Workspace Clusters (Attached or Managed Clusters). 
```

Comment:
```suggestion
When upgrading DKP, the process is different depending on whether you run a stand-alone Management Cluster, or a multi-cluster environment that includes a combination of a Management Cluster and Workspace Clusters. This can include both attached and managed clusters. 
```